core/src/components/segment/test/basic/index.html

```diff
@@ -215,15 +215,9 @@
           <ion-segment id="dynamicButtons"></ion-segment>
         </div>
 
-        <div class="ion-padding-horizontal">
-          <ion-button expand="block" onClick="toggleDisabled()">Toggle Disabled</ion-button>
-        </div>
-        <div class="ion-padding">
-          <ion-button expand="block" color="secondary" onClick="toggleValue()">Toggle Value</ion-button>
-        </div>
-        <div class="ion-padding-horizontal">
-          <ion-button expand="block" color="tertiary" onClick="toggleSwipeGesture()">Toggle Swipe Gesture</ion-button>
-        </div>
+        <button class="expand" onClick="toggleDisabled()">Toggle Disabled</button>
+        <button class="expand" color="secondary" onClick="toggleValue()">Toggle Value</button>
```

Comment:
& here for the `color` attribute.

Reply:
Removed in https://github.com/ionic-team/ionic-framework/pull/29422/commits/74d7dd7719335983a82364ba10fb54f9136522ea